cli/get.go

```diff
@@ -43,10 +43,18 @@ var getFlags = []cli.Flag{
 		Value: "10MiB",
 		Usage: "Size of each generated object. Can be a number or 10KiB/MiB/GiB. All sizes are base 2 binary.",
 	},
+	cli.StringFlag{
+		Name:  "obj.name",
+		Usage: "Specify an object name",
+	},
```

Comment:
In terms of keeping this simple - could you explain why this is needed?

It seems like it would break pretty much all scenarios since they upload multiple objects.

Reply:
This is originally written to benchmark server caching, so warp clients will read the same object multiple times, but yeah I think it is better if we make the orchestor warp client to generate a list of objects and share it with other warp clients before starting to benchmark.

Reply:
The multipart benchmark does this already: https://github.com/minio/warp/blob/1723cdbb9c71c2db26e14ed905a89da8bbd29fa0/cli/multipart.go#L94

Reply:
You may want the initiator to create the objects - otherwise they will all be fighting to create it.